src/hotkeys.cpp

```diff
@@ -54,11 +59,24 @@ static const KeycodeNames _keycode_to_name[] = {
 	{"F10", WKC_F10},
 	{"F11", WKC_F11},
 	{"F12", WKC_F12},
+	{"BACKQUOTE", WKC_BACKQUOTE},
 	{"PAUSE", WKC_PAUSE},
-	{"COMMA", WKC_COMMA},
-	{"NUM_PLUS", WKC_NUM_PLUS},
+	{"NUM_DIV", WKC_NUM_DIV},
+	{"NUM_MUL", WKC_NUM_MUL},
 	{"NUM_MINUS", WKC_NUM_MINUS},
+	{"NUM_PLUS", WKC_NUM_PLUS},
+	{"NUM_ENTER", WKC_NUM_ENTER},
+	{"NUM_DOT", WKC_NUM_DECIMAL},
+	{"/", WKC_SLASH},
+	{";", WKC_SEMICOLON},
 	{"=", WKC_EQUALS},
+	{"[", WKC_L_BRACKET},
+	{"\\", WKC_BACKSLASH},
+	{"]", WKC_R_BRACKET},
+	{"'", WKC_SINGLEQUOTE},
+	{",", WKC_COMMA},
+	{"COMMA", WKC_COMMA}, // legacy variant, should be below ","
```

Comment:
why this addition?

Reply:
`,` is just for consistency with existing `-`/`=` and new `/`/`;`/etc. And `COMMA` to support configs from previous versions. Maybe it would be better to support it in config reading instead of this hack